src/com/thizthizzydizzy/treefeller/compat/BlockRegenCompat.java

```diff
@@ -0,0 +1,48 @@
+package com.thizthizzydizzy.treefeller.compat;
+
+import com.thizthizzydizzy.treefeller.*;
+import nl.aurorion.blockregen.BlockRegen;
+import nl.aurorion.blockregen.system.preset.struct.BlockPreset;
+import nl.aurorion.blockregen.system.regeneration.struct.RegenerationProcess;
+import nl.aurorion.blockregen.system.region.struct.RegenerationRegion;
```

Comment:
Imports for other plugins should be avoided, as this will cause TreeFeller to crash if this plugin is not present.
(In some cases this requires an extra layer of separation, such as is present for the WorldGuard and JobsReborn compatibilities)